.env.example

```diff
@@ -1,15 +1,26 @@
 # GENERAL
 
-## The network used for testing purposes
+## The network used
 NETWORK_NAME="sepolia" #  ["mainnet", "sepolia", "polygon", "polygonMumbai"]
 
-# PROTOCOL CONTRACTS
+## DEPLOYMENT
+DEPLOYMENT_RPC_ENDPOINT="https://host/path"
 
+## Hex encoded private keys separated by a comma `,`
+## The first item will be the deployment wallet
+PRIVATE_KEY="0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
+
+# OSx protocol contracts
 PLUGIN_REPO_FACTORY_ADDRESS="0xA69347F49dD618cb4577670D0728684AfAa01197"
 PLUGIN_SETUP_PROCESSOR_ADDRESS="0xAc7e4fB4a2f58b7EA5516f47b6f804956Faf0134"
-
-## Hex encoded private keys separated by a comma `,`a
-PRIVATE_KEY="0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80" # Default hardhat account 0 private key. DON'T USE FOR DEPLOYMENTS
+MANAGING_DAO_ADDRESS="0x617e4fB3a2358b7EA5a16f47b6f1f4956Faf0162"
+
+# Management DAO governance plugin settings
+GOVERNANCE_PLUGIN_REPO_ADDRESS="0xf956Fa58b7EB3a2347b6f1f4f0A5a16f162617e4"
```

Comment:
Why should they insert this ? Running this script must automatically get the deployed repo's address and use it.

Reply:
// UPDATE: Now, I see that you run the installation script completely separated in which case address would be required. You can Resolve this